src/Components/Projects/Form/index.js

```diff
@@ -0,0 +1,278 @@
+      alert(error);
+    }
+  };
+
+  const cleanInputs = () => {
+    setProject(initialValue);
+    setEmployees([]);
+  };
+  const onSubmit = (e) => {
+    if (!window.location.href.includes('id')) {
+      e.preventDefault();
+      createProject(project);
+      cleanInputs();
+    } else {
+      e.preventDefault();
+      editItem(project);
+      cleanInputs();
+    }
+  };
+
+  return (
+    <>
+      <div className={formStyles.container}>
+        <h2>Form</h2>
+        <div>
+          <form className={formStyles.form} onSubmit={onSubmit}>
+            <div className={formStyles.formRaws}>
+              <label>Project Name</label>
+              <input
+                className={formStyles.inputs}
+                type="text"
+                name="name"
+                value={project.name}
+                onChange={(e) => setProject({ ...project, name: e.target.value })}
+              />
+            </div>
+            <div className={formStyles.formRaws}>
+              <label>Description</label>
+              <input
+                className={formStyles.inputs}
+                type="text"
+                name="description"
+                value={project.description}
+                onChange={(e) => setProject({ ...project, description: e.target.value })}
+              />
+            </div>
+            <div className={formStyles.formRaws}>
+              <label>Client Name</label>
+              <input
+                className={formStyles.inputs}
+                type="text"
+                name="clientName"
+                value={project.clientName}
+                onChange={(e) => setProject({ ...project, clientName: e.target.value })}
+              />
+            </div>
+            <div className={formStyles.formRaws}>
+              <label>Start Date</label>
+              <input
+                className={formStyles.inputs}
+                type="date"
+                name="startDate"
+                value={project.startDate}
+                onChange={(e) => setProject({ ...project, startDate: e.target.value })}
+              />
+            </div>
+            <div className={formStyles.formRaws}>
+              <label>End date</label>
+              <input
+                className={formStyles.inputs}
+                type="date"
+                name="endDate"
+                value={project.endDate}
+                onChange={(e) => setProject({ ...project, endDate: e.target.value })}
+              />
+            </div>
+            <div>
+              <label className={formStyles.form}> Add Employees</label>
+              {employees.map((employee, index) => (
+                <div key={index} id="employee-form">
+                  <label>Employee</label>
+                  <input
+                    className={formStyles.inputs}
+                    type="text"
+                    name="employee"
+                    value={employees.map((e) => e.employee).map((id) => id._id)}
```

Comment:
```suggestion
                    value={employees[index]._id}
```

Check if this works better. I cannot write into the employee id input. 
Also, for next week, we should work on showing the user the available employees and passing a name on the input, not an id.

Reply:
Done with "value={employees[index].employee._id}"